examples/hotrod/services/frontend/best_eta.go

```diff
@@ -76,9 +78,10 @@ func (eta *bestETA) Get(ctx context.Context, customerID string) (*Response, erro
 	}
 	eta.logger.For(ctx).Info("Found customer", zap.Any("customer", customer))
 
-	if span := opentracing.SpanFromContext(ctx); span != nil {
-		span.SetBaggageItem("customer", customer.Name)
-	}
+	span := trace.SpanFromContext(ctx)
+	bag := baggage.FromContext(ctx)
+	m := bag.Member(customer.Name)
+	span.SetAttributes(attribute.Key(m.Key()).String(m.Value()))
```

Comment:
you're doing it backwards. The original code was setting _baggage item_, making it available downstream. You're _reading_ baggage (that nobody sets) and storing it as a span attribute (which is a form of logging but has no runtime impact on the downstream).

Reply:
I see. I really stumbled upon that.